### ISSUE TITLE
Interactions with glm poisson

### ISSUE THREAD

**Issue by benjaminwnelson** [jacob-long/interactions#45]
Does this package work with a glm poisson model? I receive this error  This could cause incorrect results if data has been altered since the model was fit. You can manually provide the data to the "data =" argument.

When I enter the data argument, then it just repeats the same error.

**Comment by jacob-long** [comment]
Yes it should work with such models. The message you're seeing is not an error message and is purely informational. If the function is not producing results, then something else on your console must be reporting the cause of the error.
